### PR TITLE
Minor module changes

### PR DESCRIPTION
**Pull request by databyjp**
PR to address #1036

### CODE REVIEW

**Comment by databyjp** [comment]
@dirkkul - Just to be clear; I haven't added any tests to confirm that the new collection configs actually work as intended. Not sure if that's something you want to add in the future or not.

File: weaviate/collections/classes/config.py [hunk 1]
```diff
@@ -684,23 +691,31 @@ def palm(
 
     @staticmethod
     def aws(
-        model: str,
         region: str,
+        service: Union[AWSService, str] = "bedrock",
+        model: Optional[str] = None,
```
Comment (dirkkul) [inline weaviate/collections/classes/config.py:696]: this is sadly a breaking change, because of the order :(

Comment (dirkkul) [reply weaviate/collections/classes/config.py:696]: if people have positional arguments in there, it would break

Comment (dirkkul) [reply weaviate/collections/classes/config.py:696]: ah bummer - if we need to break anyways, let's add `*` as a first parameter then all parameters are kwargs

Comment (databyjp) [reply weaviate/collections/classes/config.py:694]: [Found this](https://github.com/weaviate/weaviate-python-client/blob/9480b60ca3c3ae66df9aaf52a09bffcf5cbc600b/weaviate/collections/classes/config_vectorizers.py#L547) while looking for the right place to edit. So maybe we could follow the style.

Comment (dirkkul) [reply weaviate/collections/classes/config.py:694]: I see ok - I would still be fine with changing it kwargs-only

**Review by dirkkul — APPROVED** [review]

**Pull request merged**